examples/webgl_loader_gltf.html

```diff
@@ -382,6 +383,14 @@
 					extensions: ["glTF", "glTF-MaterialsCommon", "glTF-Binary"]
 				},
 				{
+					name : "BoomBox (PBR)", url : "./models/gltf/BoomBox/%s/BoomBox.gltf",
+					cameraPos: new THREE.Vector3(2, 1, 3),
+					objectRotation: new THREE.Euler(0, Math.PI, 0),
+					addLights:true,
+					extensions: ["glTF"]
```

Comment:
Well, somehow `gltf-Binary` doesn't work for BoomBox...
Could you take a look closer later?